test/integration-new/README.md

```diff
@@ -54,6 +54,43 @@ In order to test a custom image you need pass the following tags along with the
 
 *IMPORTANT*: Should use an IPv6 cluster with Prefix Delegation enabled. VPC CNI only supports IPv6 mode with Prefix Delegation.
 
+### Multus tests
+These tests require multus to be deployed to your cluster using the [manifest](https://raw.githubusercontent.com/aws/amazon-vpc-cni-k8s/master/config/multus/v3.7.2-eksbuild.2/aws-k8s-multus.yaml) file. Instead test can be triggered by running `run-multus-tests.sh` located under scripts directory. This script installs the latest multus manifest first and then runs the the ginkgo test suite. 
+
+```
+KUBE_CONFIG_PATH=/Users/cgadgil/.kube/config CLUSTER_NAME=eks-MultusInfra REGION=us-west-2 ./scripts/run-multus-tests.sh
+
+Running tests with the following variables
+KUBE_CONFIG_PATH:  /Users/cgadgil/.kube/config
+CLUSTER_NAME: eks-MultusInfra
+REGION: us-west-2
+ENDPOINT: 
+loading cluster details eks-MultusInfra
+Installing latest multus manifest with tag: v3.7.2-eksbuild.2
```

Comment:
Even here, it would be good if we can get the latest tag from GitHub rather than hardcoding.

Reply:
we dont push multus tag on github. so I have added it as argument, if not provided then use some default

Reply:
for Iteration2, will check how we can pull latest tags from internal gitfarm

Reply:
sure sounds good.